### ISSUE TITLE
Form-fields of Inaccessible form is still showing the label

### ISSUE THREAD

**Issue by Madhav4466** [tekvision/tekvision.github.io#2]
To show differentiation with the accessible form the other one should be purely inaccessible. Need to Remove the aria-label attribute from all form fields of [inaccessible form.](https://github.com/tekvision/tekvision.github.io/blob/master/Widget%20Pages/Form/Forms/forms-page.html)

**Comment by sidnc86** [comment]
Please provide path of files to work on.

**Comment by Madhav4466** [comment]
> Please provide path of files to work on.

Actually, I have already given a path at the last word of the description via link.
If you see "Inaccessible form" which is a link and it redirects to the file which is I am talking about.

Thanks
